plugins/woocommerce/src/Blocks/BlockTypes/ProductCollection/QueryBuilder.php

```diff
@@ -176,8 +176,8 @@ public function get_final_frontend_query( $collection_args, $query, $page = 1, $
 			$common_query_values,
 			array(
 				'on_sale'             => $is_on_sale,
-				'stock_status'        => $query['woocommerceStockStatus'],
-				'orderby'             => $query['orderBy'],
+				'stock_status'        => $query['woocommerceStockStatus'] ?? array_keys( wc_get_product_stock_status_options() ),
+				'orderby'             => $query['orderBy'] ?? 'title',
```

Comment:
Nitpick: I'd suggest extracting these to variables to keep the same convention as other parameters a bit higher (lines 161-166):

```
$stock_status            = $query['woocommerceStockStatus'] ?? array_keys( wc_get_product_stock_status_options() );
$order_by                = $query['orderBy'] ?? 'title';
```

```suggestion
				'stock_status'        => $stock_status
				'orderby'             => $order_by,
```